Exercise_2.py

```diff
@@ -1,3 +1,8 @@
+# Time complexity for :
+   # push operation = O(n)
```

Comment:
Push() is also having O(1) time complexity as you are adding node at head itself.